app/lib/dom.ts

```diff
@@ -0,0 +1,33 @@
+export function isInSubtreeOf(element: Element, target: Element) {
```

Comment:
would Node.contains() work? https://developer.mozilla.org/en-US/docs/Web/API/Node/contains

Reply:
Ah cool, TIL!

Deleted this function since it's easy enough to inline `element === target || element.contains(target)`